SCons/Tool/gcc.py

```diff
@@ -57,6 +57,9 @@ def generate(env):
     if version:
         env['CCVERSION'] = version
 
+    env['CCDEPFLAGS'] = '.MMD -MF ${TARGET}.d'
```

Comment:
should be `-MMD` and this seems like a lot of extra whitespace

Reply:
fix just pushed